app/domain/models/impl/SimpleCategoryModel.scala

```diff
@@ -1,29 +1,52 @@
 package domain.models.impl
```

Comment:
@Bukarachka Let's not use Impl in project. Also, it's nice to package classes by functionality, not by it's type.